src/renderer/src/routes/pages/BuyAndSellTokens/BuyTokensWalletItem.tsx

```diff
@@ -0,0 +1,62 @@
+import WalletIcon from '@renderer/assets/images/wallet-icon.svg?react'
+import { Accordion } from '@renderer/components/Accordion'
+import { Loader } from '@renderer/components/Loader'
+import { Separator } from '@renderer/components/Separator'
+import { Tooltip } from '@renderer/components/Tooltip'
+import { NumberHelper } from '@renderer/helpers/NumberHelper'
+import { useBalances } from '@renderer/hooks/useBalances'
+import { useCurrencySelector } from '@renderer/hooks/useSettingsSelector'
+import { IWalletState } from '@shared/@types/store'
+
+import { BuyTokensAccountItem } from './BuyTokensAccountItem'
+
+type TProps = {
+  isDefaultOpened: boolean
+  wallet: IWalletState
+}
+
+export const BuyTokensWalletItem = ({ isDefaultOpened, wallet }: TProps) => {
+  const { id, accounts } = wallet
+  const balances = useBalances(accounts)
+  const { currency } = useCurrencySelector()
+  const total = NumberHelper.currency(balances.exchangeTotal, currency.label)
+
+  return (
+    <Accordion.Root className="flex flex-col" type="multiple" defaultValue={isDefaultOpened ? [id] : []}>
```

Comment:
How I said above, it does not make sense, this component should be only the `Accordion.Item`